content/installation/net/install/Installer-CommandLine-Options.md

```diff
@@ -0,0 +1,38 @@
+<!--
+title: "Contrast .NET Agent Installer Command Line Options"
+description: "Contrast .NET agent installer command line options"
+tags: "installation agent .NET windows silent unattended installer setup"
+-->
+
+## .NET Agent Installer Command Line Options
+
+The .NET agent installer supports additional options when invoked from the command line.
+
+### Silent Installation
+
+The .NET agent installer supports standard silent installation options. These options tell the installer to run silently and unattended, which means that the installer don't require your interaction and don't present the installer's UI.
+
+- Install: `ContrastSetup.exe -s -norestart`
+
+### Uninstalling and Repairing
+
+The .NET agent installation can be uninstall or repaired using standard Windows features (e.g. the Programs and Features Control Panel, Powershell, etc.). However, it may be desirable to use the .NET agent installer to perform these actions instead.
```

Comment:
"may be desirable" sounds a bit vague.  I would just say something like "However for automation and scripted scenarios you may also perform these actions from the command line".